packages/usa-date-picker/src/index.js

```diff
@@ -734,6 +734,8 @@ const ariaDisable = (el) => {
 const enable = (el) => {
   const { externalInputEl, toggleBtnEl } = getDatePickerContext(el);
 
+  el.removeAttribute("readonly");
+  externalInputEl.removeAttribute("aria-disabled");
   toggleBtnEl.disabled = false;
   externalInputEl.disabled = false;
```

Comment:
**issue:** Similar to the `ariaDisabled` function, we'll want to update `el` to `externalInputEl`.

**issue:** We removing `disabled` from the `toggleBtnEl` but not `aria-disabled`. We should make sure to remove this as well.

I'm also grouping element changes in this suggestion:

```suggestion
  toggleBtnEl.disabled = false;
  toggleBtnEl.removeAttribute("aria-disabled");
  
  externalInputEl.disabled = false;
  externalInputEl.removeAttribute("aria-disabled");
  externalInputEl.removeAttribute("readonly");
```

Reply:
Resolved in [18e578c](https://github.com/uswds/uswds/pull/6013/commits/18e578c8c71e20ea38cf57edc3e0fdb130e76c0f)